samples/README.md

```diff
@@ -41,3 +41,21 @@ This section lists how-to guides for the most commonly used APIs and most common
 #### General How-To Guides
 
 - How to configure, access, and analyze **logging** information (TODO)
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+If you'd like to contribute to this library, please read the [contributing guide] [contributing_guide] to learn more about how to build and test the code.
```

Comment:
```suggestion
If you'd like to contribute to this library, please read the [contributing guide][contributing_guide] to learn more about how to build and test the code.
```

Reply:
Otherwise, it loooks like this:
![image](https://user-images.githubusercontent.com/6527137/236040615-76139530-e9d0-4626-88a2-ef3ea50ddd7b.png)

Reply:
Good catch.  This came verbatim from compliance team.  I made the fix